documentation/modules/configuring/con-config-mirrormaker2-replication.adoc

```diff
@@ -81,6 +85,30 @@ Both topics enable replication to be fully restored from the correct offset posi
 
 MirrorMaker 2.0 uses its `MirrorCheckpointConnector` to emit _checkpoints_ for offset tracking.
 
+== Synchronizing consumer group offsets
+
+The `__consumer_offsets` topic stores information on committed offsets, for each consumer group.
+Offset synchronization periodically transfers the consumer offsets for the consumer groups of a source cluster into the consumer offsets topic of a target cluster.
+
+Offset synchronization is particularly useful in an _active/passive_ configuration.
+If the active cluster goes down, consumer applications can switch to the passive (standby) cluster and pick up from the last transferred offset position.
+
+To use topic offset synchronization:
+
+* Enable the synchronization by adding `sync.group.offsets.enabled` to the checkpoint connector configuration of the `KafkaMirrorMaker2` resource,
+and setting the property to `true`. Synchronization is disabled by default.
+* Add the `IdentityReplicationPolicy` to the source and checkpoint connector configuration so that topics in the target cluster retain their original names.
+
+For topic offset synchronization to work, consumer groups in the target cluster cannot use the same ids as groups in the source cluster.
+
+If enabled, the synchronization of offsets from the source cluster is made periodically.
+You can change the frequency by adding `sync.group.offsets.interval.seconds` and `emit.checkpoints.interval.seconds` to the checkpoint connector configuration.
+The properties specify the frequency in seconds that the consumer group offsets are synchronized, and the frequency of checkpoints emitted for offset tracking.
+The default for both properties is 60 seconds.
+You might also want to change the frequency of checks for new consumer groups using the `refresh.groups.interval.seconds` property, which is performed every 10 minutes by default.
```

Comment:
TBH, I'm not sure we should suggest the change. We do not really know what impact these have on performance etc. So I think saying _You can also change ..._ would be better.

Reply:
Changed to: _You can also change the frequency of checks for new consumer groups using the `refresh.groups.interval.seconds` property, which is performed every 10 minutes by default._